<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-6NURCCG" failures="1" tests="5" name="c11Practice.executionOrder.PriorityDisabledDependencyTesting" time="0.012" errors="0" timestamp="2022-09-06T15:37:47 EDT" skipped="3">
  <testcase classname="c11Practice.executionOrder.PriorityDisabledDependencyTesting" name="practiceReporting" time="0.009"/>
  <system-out/>
  <testcase classname="c11Practice.executionOrder.PriorityDisabledDependencyTesting" name="loginWithValidCredentials" time="0.002">
    <failure message="PurposelyFailing" type="java.lang.AssertionError">
        <![CDATA[java.lang.AssertionError: PurposelyFailing
at org.testng.Assert.fail(Assert.java:97)
at c11Practice.executionOrder.PriorityDisabledDependencyTesting.loginWithValidCredentials(PriorityDisabledDependencyTesting.java:11)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
    </failure>
  </testcase> <!-- loginWithValidCredentials -->
  <system-out/>
  <testcase classname="c11Practice.executionOrder.PriorityDisabledDependencyTesting" name="logoVerification" time="0.001">
    <skipped/>
  </testcase> <!-- logoVerification -->
  <system-out/>
  <testcase classname="c11Practice.executionOrder.PriorityDisabledDependencyTesting" name="logOutFromHrmApplication" time="0.000">
    <skipped/>
  </testcase> <!-- logOutFromHrmApplication -->
  <system-out/>
  <testcase classname="c11Practice.executionOrder.PriorityDisabledDependencyTesting" name="verifyAvailableTabsInHomePage">
    <skipped/>
  </testcase> <!-- verifyAvailableTabsInHomePage -->
  <system-out/>
</testsuite> <!-- c11Practice.executionOrder.PriorityDisabledDependencyTesting -->
